general-concepts/ebuild-revisions/text.xml

```diff
@@ -95,7 +99,11 @@ of thumb could be used as a guideline:
     is expected soon),
   </li>
   <li>
-    adding a missing build-time dependency that caused a build failure,
+    adding a missing build-time dependency that caused a build failure that
+    is not a runtime dependency (e.g. a missing library dependency which
+    happened to fail in src_configure still needs a new revision because
+    it would manifest as an automagic dependency if a user had the package
+    installed already),
```

Comment:
[For some reason, my previous comment disappeared.]

I think these examples aren't really necessary, and may distract from more important things (i.e. make the section too long so that people won't read it thoroughly any more). I also dislike the double "that". How about the following:
```
  <li>
    adding a missing build-time dependency that caused a build failure
    (unless it is also a runtime dependency),
  </li>
```

(Thinking about it, we probably should spell "build-time" and "run-time" consistently, i.e. both with or without a hyphen.)

Reply:
Wonder if there's room for a whole new section at the bottom of the page; concrete examples with rationale explained in detail (like it's not **really** explained why touching anything-RDEPEND always requires a revbump - at least in this page)

Or if such information(s) already exists, you could maybe add some sort of `[1]` internal links to these short sections.

Just an idea.

Reply:
Yeah, I think that's a good point, although I'm not sure I've got it in me right now to start the new section. It's definitely something we should do though.